lib/src/controller/flutter_pagination_controller.dart

```diff
@@ -93,4 +86,72 @@ class FlutterPaginationController<ItemType, PM extends PaginationMethod,
       }
     }
   }
+
+  /// Fetches the first page of data.
+  @override
+  Future<void> getFirst() async {
+    isProcessing.value = true;
+    _state = await handlePagination(firstPagePointer, true);
+    isProcessing.value = false;
```

Comment:
There is no error handling, isProcessing will hang at true if an exception occurs in the asynchronous call